backend/spec/system/backoffice/project_developers_spec.rb

```diff
@@ -74,11 +91,117 @@
+        select "Spanish", from: t("simple_form.labels.account.language")
+        click_on t("backoffice.common.save")
+        expect(page).to have_text(t("backoffice.messages.success_update", model: t("backoffice.common.project_developer")))
+        expect(approved_pd.account.reload.language).to eq("es")
+      end
+    end
+
+    context "approval status section" do
+      before { within_sidebar { click_on t("backoffice.account.approval_status") } }
+
+      it "can update approval status" do
+        expect(page).to have_select(t("simple_form.labels.account.review_status"), selected: ReviewStatus.find("approved").name)
+        select "Unapproved", from: t("simple_form.labels.account.review_status")
+        click_on t("backoffice.common.save")
+        expect(page).to have_text(t("backoffice.messages.success_update", model: t("backoffice.common.project_developer")))
+        expect(approved_pd.account.reload.review_status).to eq("unapproved")
+      end
+    end
+
+    context "profile section" do
+      context "when content is correct" do
+        it "can update profile information" do
+          attach_file t("simple_form.labels.account.picture"), Rails.root.join("spec/fixtures/files/picture_2.jpg")
+          fill_in t("simple_form.labels.account.name"), with: "New profile name"
+          select t("enums.project_developer_type.academic.name"), from: t("simple_form.labels.project_developer.project_developer_type")
+          fill_in t("simple_form.labels.project_developer.entity_legal_registration_number"), with: "1111111111"
+          fill_in t("simple_form.labels.account.about"), with: "New about description"
+          fill_in t("simple_form.labels.defaults.mission"), with: "New mission"
+          fill_in t("simple_form.labels.account.website"), with: "http://new-example.com"
+          fill_in t("simple_form.labels.account.contact_phone"), with: "+48123123123"
+          fill_in t("simple_form.labels.account.contact_email"), with: "newemail@example.com"
+          fill_in "Linkedin", with: "https://linkedin.com/new-profile"
+          fill_in "Instagram", with: "https://instagram.com/new-profile"
+          fill_in "Facebook", with: "https://facebook.com/new-profile"
+          fill_in "Twitter", with: "https://twitter.com/new-profile"
+          check t("enums.category.tourism-and-recreation.name")
+          uncheck t("enums.category.forestry-and-agroforestry.name")
+          check t("enums.impact.community.name")
+          uncheck t("enums.impact.water.name")
+          uncheck t("enums.impact.climate.name")
+          check t("enums.mosaic.amazon-heart.name")
+          click_on t("backoffice.common.save")
+
+          expect(page).to have_text(t("backoffice.messages.success_update", model: t("backoffice.common.project_developer")))
+          approved_pd.reload
+          expect(approved_pd.account.picture.filename.to_s).to eq("picture_2.jpg")
```

Comment:
This should be probably replaced at future by some page detail for updated PD -- which is basically true even for other tests

Reply:
I don't understand :)

Reply:
Ha :), sry. Preferred use of expect at system tests should be check that some page contains some expected values. I believe that it is more appropriate after PD gets updated to go to page with detail of PD and check that values are correct there instead of checking ActiveRecord itself. Ofc we dont have such page right now and I dont think we need to be super strict with this so just my opinion on this topic ;)

Reply:
Yeah. I was thinking if to check the inputs fields, after the page is reloaded, but thought this will be faster. We won't have details page. (I saw that message sent by email went to main thread)

Reply:
Maybe it is worth to refactor this to check those inputs and checkboxes if those are still correctly selected.